mathics/eval/plot.py

```diff
@@ -102,15 +100,10 @@ def quiet_f(*args):
 
             return quiet_f
     expr: Optional[Type[BaseElement]] = Expression(SymbolN, expr).evaluate(evaluation)
-    quiet_expr = Expression(
```

Comment:
This came up when I run the doctests with the other changes: In WMA, when the points are evaluated, `*Plot` hides all the warnings, not just "Power::infy". This could go in a different PR, but the natural tests needs of the changes introduced here. (I also tested with functions like `F[x_]:=(Open["nonexistingfile"];x^2)` but it is just not so nice.